src/main/java/com/rocketden/main/dto/game/GameMapper.java

```diff
@@ -0,0 +1,51 @@
+package com.rocketden.main.dto.game;
+
+import com.rocketden.main.dto.room.RoomMapper;
+import com.rocketden.main.game_object.Game;
+import com.rocketden.main.game_object.Player;
+import com.rocketden.main.model.Room;
+import com.rocketden.main.model.User;
+
+public class GameMapper {
+
+    protected GameMapper() {}
+
+    public static GameDto toDto(Game game) {
+        if (game == null) {
+            return null;
+        }
+
+        // For now, just include the room info in the GameDto
+        GameDto gameDto = new GameDto();
+        gameDto.setRoomDto(RoomMapper.toDto(game.getRoom()));
+
+        return gameDto;
+    }
+
+    public static Game fromRoom(Room room) {
+        if (room == null) {
+            return null;
+        }
+
+        Game game = new Game();
+        game.setRoom(room);
+
+        for (User user : room.getUsers()) {
+            Player player = playerFromUser(user);
+            game.getPlayers().put(user.getNickname(), player);
```

Comment:
nit: Save `game.getPlayers()` as a variable outside of the for loop, like in a value called `players` or `playerList`, then use `put` on this variable. Not sure if this is actually more efficient, but then `getPlayers()` is only called once.